### PR TITLE
Hello Microverse Linter and Project Setup

### PR DESCRIPTION
**Pull request by ohyajk**
I added Linters in the main branch and created an additional branch for HTML & CSS files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
-# HelloMicroverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HelloMicroverse
+
+> This is a Linter Setup Project to catch syntax errors in HTML & CSS.
+
+
```
Comment (bobb-Rob) [inline README.md:7]: - [x] I notice that your linter checks are not running as Github actions on this pull request. kindly ensure that linter checks are running on GitHub as expected. Kindly fix that and resubmit.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @ohyajk, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ohyajk) [reply README.md:7]: @bobb-Rob Linter show 4 green ticks. I don't understand what are you trying to say. can you show a youtube video of error?
linters

**Review by Lameck1 — COMMENTED** [review]
Hello, @ohyajk 👋,

_This is an additional review as a result of the second opinion request._

It is true that GitHub actions are not running. The screenshots below verify this:

| This first screenshot shows that 0 checks ran in your last commit |
| --- |
| Screenshot 2022-10-26 071841 | 

| This second screenshot shows that there is no green checkmark in the last commit, therefore, no checks ran |
| --- |
| Screenshot 2022-10-26 072500 |

In code reviews, it is only the last commit that is considered. Your linter check must be running all the time. In this case, I suspect that the GitHub workflow for this repo must have been disabled. [Please go through this guide and learn how you can reenable the workflows](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#:~:text=workflow%20can%20run.-,Managing%20GitHub%20Actions%20permissions%20for%20your%20repository,Click%20Save.,-Allowing%20select%20actions).

One more thing that @bobb-Rob failed to point out is your commit messages.

  **_If you want to write a git commit message properly, you should use the imperative mood. This means you need to eliminate the temptation to use gerunds or past tense in your subject lines. Don't write a git commit subject line that talks about what you did, or what you are doing. Instead, describe what was done._**
    
  Another rule for commit messages:    
  - The first word must be capitalized. For example, instead of **"add readme"**, it should be **"Add readme"**. Besides this, the message should be meaningful. Referring to your own commit message, instead of writing **"html add"**, you should have written **"Add HTML"**. HTML is an acronym and should be in all UPPERCASE. Your second commit message should have been **"Add CSS"**. Your last commit message could have been **"Update Readme"**
  
  [Consider reading this 2015 article on commit messages](https://www.codelord.net/2015/03/16/bad-commit-messages-hall-of-shame/#:~:text=%E2%80%9Cbug%20fix%E2%80%9D%2C%20%E2%80%9Cmore,well%20supply%20no%20commit%20message.), it is relevant and can help.
  
  NOTE: You are NOT expected to refactor your previous commit messages. I am just pointing this out to help you write better ones in the future.
  

Please enable GitHub action in this repo, ensure all linters are running and are green, then ask for another code review.

Keep rocking 🚀!

**Comment by ohyajk** [comment]
I done as the link say which you provided but green tick is not showing. You are not helping me

**Comment by ohyajk** [comment]
just wasting my reviews

**Pull request closed without merging**